apis/go.mod

```diff
@@ -3,25 +3,26 @@ module github.com/openshift/hive/apis
 go 1.17
 
 require (
-	github.com/openshift/api v0.0.0-20220427141643-6e7069f2fba2
-	k8s.io/api v0.23.0
-	k8s.io/apimachinery v0.23.0
+	github.com/openshift/api v0.0.0-20220222102030-354aa98a475c
```

Comment:
The version of `github.com/openshift/api` we use here will need to be synced with the top level `go.mod` file. I thought we had a card to add a test for this but I couldn't find one so I made https://issues.redhat.com/browse/HIVE-1941.

Reply:
I think `make vendor` or other make command should fix such issues.

Reply:
https://github.com/openshift/hive/pull/1810 will enforce :)